src/tools/image2heif/main.c

```diff
@@ -1786,30 +1812,33 @@ static int  i2h_universe_item_handle_picture      (UNIVERSE* u, ITEM* item){
     
     /**
      * MIDDLEEND:
-     *    1. (Optional) Encode thumbnail tile.
-     *    2. Encode grid tiles.
-     *    3. Flush encoder.
-     *    4. Split/duplicate single list of packets between thumbnail and grid tiles.
-     *    5. Create new items and fill them.
-     *    6. If this item is not single-tile,
-     *        6.1 Transmute to "grid".
-     *        6.2 Compute grid references.
+     *    1. Configure encoder.
+     *    2. (Optional) Encode thumbnail tile.
+     *    3. Encode grid tiles.
+     *    4. Flush encoder.
+     *    5. Release encoder.
+     *    6. Split/duplicate single list of packets between thumbnail and grid tiles.
+     *    7. Create new items and fill them.
+     *    8. If this item is not single-tile,
+     *        8.1 Transmute to "grid".
+     *        8.2 Compute grid references.
      */
     
-    /**
-     * Push encoded tiles into the encoder.
```

Comment:
Might want to hold on to the comments no ?

Reply:
The way the middle-end is written, with descriptive function names and error messages, it's somewhat redundant.